2022-02-28 13:05:15,230 INFO    MainThread:620424 [wandb_setup.py:_flush():71] setting env: {}
2022-02-28 13:05:15,230 INFO    MainThread:620424 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-28 13:05:15,230 INFO    MainThread:620424 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-28 13:05:15,230 INFO    MainThread:620424 [wandb_init.py:_log_setup():371] Logging user logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220228_130515-2tk370pc/logs/debug.log
2022-02-28 13:05:15,230 INFO    MainThread:620424 [wandb_init.py:_log_setup():372] Logging internal logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220228_130515-2tk370pc/logs/debug-internal.log
2022-02-28 13:05:15,230 INFO    MainThread:620424 [wandb_init.py:init():404] calling init triggers
2022-02-28 13:05:15,230 INFO    MainThread:620424 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-28 13:05:15,230 INFO    MainThread:620424 [wandb_init.py:init():460] starting backend
2022-02-28 13:05:15,230 INFO    MainThread:620424 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-28 13:05:15,236 INFO    MainThread:620424 [backend.py:ensure_launched():216] starting backend process...
2022-02-28 13:05:15,240 INFO    MainThread:620424 [backend.py:ensure_launched():222] started backend process with pid: 620433
2022-02-28 13:05:15,241 INFO    MainThread:620424 [wandb_init.py:init():469] backend started and connected
2022-02-28 13:05:15,242 INFO    MainThread:620424 [wandb_init.py:init():533] updated telemetry
2022-02-28 13:05:15,243 INFO    MainThread:620424 [wandb_init.py:init():563] communicating current version
2022-02-28 13:05:15,698 INFO    MainThread:620424 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-28 13:05:15,698 INFO    MainThread:620424 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-28 13:05:16,000 INFO    MainThread:620424 [wandb_init.py:init():606] starting run threads in backend
2022-02-28 13:05:18,622 INFO    MainThread:620424 [wandb_run.py:_console_start():1810] atexit reg
2022-02-28 13:05:18,622 INFO    MainThread:620424 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-28 13:05:18,623 INFO    MainThread:620424 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-28 13:05:18,624 INFO    MainThread:620424 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-28 13:05:18,624 INFO    MainThread:620424 [wandb_init.py:init():633] run started, returning control to user process
2022-02-28 13:05:34,324 INFO    MainThread:620424 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-28 13:05:34,325 INFO    MainThread:620424 [wandb_run.py:_restore():1752] restore
2022-02-28 13:05:37,163 INFO    MainThread:620424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 681
}

2022-02-28 13:05:37,436 INFO    MainThread:620424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 681
}

2022-02-28 13:05:38,018 INFO    MainThread:620424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 6211
}

2022-02-28 13:05:38,120 INFO    MainThread:620424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 15810
}

2022-02-28 13:05:38,222 INFO    MainThread:620424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 15810
}

2022-02-28 13:05:38,325 INFO    MainThread:620424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 15810
}

2022-02-28 13:05:38,427 INFO    MainThread:620424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15810
  total_bytes: 15810
}

2022-02-28 13:05:38,530 INFO    MainThread:620424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15810
  total_bytes: 15810
}

2022-02-28 13:05:38,632 INFO    MainThread:620424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15810
  total_bytes: 15810
}

2022-02-28 13:05:38,734 INFO    MainThread:620424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15810
  total_bytes: 15810
}

2022-02-28 13:05:38,837 INFO    MainThread:620424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15810
  total_bytes: 15810
}

2022-02-28 13:05:38,939 INFO    MainThread:620424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15810
  total_bytes: 15810
}

2022-02-28 13:05:39,223 INFO    MainThread:620424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15810
  total_bytes: 15810
}

2022-02-28 13:05:39,970 INFO    MainThread:620424 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15810
  total_bytes: 15810
}
local_info {
}

2022-02-28 13:05:41,037 INFO    MainThread:620424 [wandb_run.py:_append_history():2130] rendering history
2022-02-28 13:05:41,037 INFO    MainThread:620424 [wandb_run.py:_append_summary():2085] rendering summary
2022-02-28 13:05:41,037 INFO    MainThread:620424 [wandb_run.py:_append_files():2180] logging synced files
